policy-controller/grpc/src/outbound.rs

```diff
@@ -154,7 +151,7 @@ where
             .index
             .watch_outbound_policy(service)
             .await
-            .map_err(|e| tonic::Status::internal(format!("lookup failed: {}", e)))?
+            .map_err(|e| tonic::Status::internal(format!("lookup failed: {e}",)))?
```

Comment:
super nitpicky but could probably remove the comma.

```suggestion
            .map_err(|e| tonic::Status::internal(format!("lookup failed: {e}")))?
```